content/docs/gke/authentication.md

```diff
@@ -0,0 +1,175 @@
++++
+title = "Authenticating Kubeflow to GCP"
+description = "Troubleshooting guide for authentication and authorization to GCP"
+weight = 4
++++
+
+
+# Authenticating gcloud
+
+[The `gcloud` tool](https://cloud.google.com/sdk/gcloud/) is used to interact with Google Cloud Platform (GCP) over the command line. 
+It can be used to [set up Google Kubernetes Engine (GKE) clusters](https://cloud.google.com/sdk/gcloud/reference/container/clusters/create), 
+and interact with other Google services.
+
+### Logging in
+
+There are two ways to authenticate the gcloud command:
+
+
+- A **service account** is an account set up within your GCP project. Authentication is handled by 
```

Comment:
Please clarify who needs to perform the following action:

`Authentication is handled by downloading ...`

Change the sentence to active voice and let the reader know if they have to do this or if some script or other does it.